apollo-test/src/main/java/com/spotify/apollo/test/ServiceHelper.java

```diff
@@ -478,7 +480,18 @@ private void start(int timeoutSeconds, final String... args) throws InterruptedE
             .withModule(HttpClientModule.create())
             .withModule(
                 ForwardingStubClientModule
-                    .create(forwardNonStubbedRequests, stubClient.asRequestAwareClient()));
+                    .create(forwardNonStubbedRequests, stubClient.asRequestAwareClient()))
+            .withModule(new AbstractApolloModule() {
```

Comment:
This should probably not be inline like this.

Maybe make the `MetaModule` public with some api for creating it with the assembly name?